setup.py

```diff
@@ -73,8 +73,6 @@ def run(self):
 if platform.machine() == 'ppc64le':
     extra_link_args += ['-pthread']
 
-build_ext = cmdclass.get('build_ext', build_ext)
```

Comment:
This came from the vendored Versioneer's `cmdclass`, which I think raises two questions:

- The vendored Versioneer we have is still using `distutils`. What's the right course of action here? Should we update the vendored version, or try to un-vendor it?
- Is `setuptools.command.build_ext.build_ext` the same thing as returned by `cmdclass.get('build_ext', build_ext)` when `cmdclass` is Versioneer's `cmdclass`?

(I also think there's the question of whether continuing to use Versioneer is the right way to go, but I *definitely* don't want to muddy the waters with that right now)

Reply:
I reverted it. And will address versioneer issue later.